syntax = "proto2";

message GetActionNamesResp {
	repeated string name_array = 1;
}
